packages/client/test/rpc/engine/forkchoiceUpdatedV1.spec.ts

```diff
@@ -271,7 +271,7 @@ tape(`${method}: latest block after reorg`, async (t) => {
   ])
 
   expectRes = (res: any) => {
-    t.equal(res.body.result.payloadStatus.latestValidHash, blocks[1].blockHash)
+    t.equal(res.body.error.code, -32602)
```

Comment:
Note; this FCU payload is indeed invalid, because `headBlock` has block number 1, while  `safeBlock` has block number 2 (and `finalBlock` also)